scipy/stats/_continuous_distns.py

```diff
@@ -40,11 +40,20 @@ class ksone_gen(rv_continuous):
     %(default)s
 
     """
+    def _pdf(self, x, n):
```

Comment:
it would be nice to expand the description of ksone_gen and kstwobign (potentially reuse text from special/add_newdocs.py ), maybe also reference to the implemention of the functions in scipy.special